bin/kuzzle-start.js

```diff
@@ -41,6 +39,14 @@ module.exports = function () {
 ██     KUZZLE ` + (kuzzle.isServer ? 'SERVER' : 'WORKER') + ` STARTED      ██
 ████████████████████████████████████`);
     })
+    .then(() => {
+      /*
+       Waits for at least one write worker to be connected to the server before trying to use them
+       */
+      return kuzzle.services.list.broker.waitForListeners(kuzzle.config.queues.workerWriteTaskQueue);
+    })
+    .then(() => { return kuzzle.cleanDb(); })
+    .then(() => { return kuzzle.prepareDb(); })
```

Comment:
I'm not sure about running cleanDb after the message "Kuzzle is started". Because just after, we have to wait for workers